content/200-concepts/100-components/01-prisma-schema/06-relations/400-self-relations.mdx

```diff
@@ -283,8 +283,8 @@ A many-to-many self-relation looks as follows:
 model User {
   id         Int     @id @default(autoincrement())
   name       String?
-  followedBy User[]  @relation("UserFollows", references: [id])
-  following  User[]  @relation("UserFollows", references: [id])
+  followedBy User[]  @relation("UserFollows")
+  following  User[]  @relation("UserFollows")
```

Comment:
I think this was the only _actual_ broken implicit m:n example in the whole docs that was using `references` 😆

Reply:
Ah so there was one! Didn't notice any on my very quick check before!

Reply:
It was a weird one with the self relation.